code/modules/antagonists/traitor/objectives/destroy_item.dm

```diff
@@ -13,16 +13,28 @@
 	abstract_type = /datum/traitor_objective/destroy_item
 
 /datum/traitor_objective/destroy_item/low_risk
-	progression_minimum = 10 MINUTES
+	progression_minimum = 5 MINUTES
+	progression_maximum = 25 MINUTES
+	progression_reward = list(5 MINUTES, 10 MINUTES)
+	telecrystal_reward = list(1, 2)
+
+	possible_items = list(
+		/datum/objective_item/steal/traitor/cargo_budget,
+		/datum/objective_item/steal/traitor/clown_shoes,
+	)
+
+/datum/traitor_objective/destroy_item/high_risk
+	progression_minimum = 20 MINUTES
 	progression_maximum = 35 MINUTES
 	progression_reward = list(5 MINUTES, 10 MINUTES)
-	telecrystal_reward = 1
+	telecrystal_reward = list(4,5)
 
 	possible_items = list(
 		/datum/objective_item/steal/traitor/bartender_shotgun,
 		/datum/objective_item/steal/traitor/fireaxe,
 		/datum/objective_item/steal/traitor/nullrod,
 		/datum/objective_item/steal/traitor/big_crowbar,
```

Comment:
These should be moved to low risk. They're as risky to do as the clown shoes and the cargo budget.